app/src/main/java/com/hover/stax/home/MainActivity.kt

```diff
@@ -227,6 +228,10 @@ class MainActivity : AbstractRequestActivity(), BalancesViewModel.RunBalanceList
         Timber.e("uuid? %s", data?.extras?.getString("uuid"))
 
         when {
+            resultCode == RESULT_CANCELED && requestCode == Constants.TRANSFER_REQUEST -> {
```

Comment:
This still isn't correct. It would override the scheduling case and not apply to balances. As stated before, it should be integrated into the showPopUpTransactionDetailsIfRequired method.

Reply:
Done, thanks!